app/src/main/java/com/android/sample/model/recipe/CreateRecipeViewModel.kt

```diff
@@ -247,18 +278,34 @@ class CreateRecipeViewModel(private val repository: FirestoreRecipesRepository)
     val newUid = repository.getNewUid()
     require(newUid.isNotBlank()) { "Recipe ID must not be blank." }
```

Comment:
After some reflection, it appears this check might be unnecessary. Since `repository.getNewUid()` calls `db.collection(FIRESTORE_COLLECTION_NAME).document().id`, which, according to the documentation, reliably returns a non-null object, we can safely remove this check.